1-js/02-first-steps/11-logical-operators/article.md

```diff
@@ -1,289 +1,290 @@
-# Logical operators
+# Տրամաբանական օպերատորներ (Logical operators)
 
-There are four logical operators in JavaScript: `||` (OR), `&&` (AND), `!` (NOT), `??` (Nullish Coalescing). Here we cover the first three, the `??` operator is in the next article.
+JavaScript-ում կան չորս տրամաբանական օպերատորներ՝ `||` (ԿԱՄ), `&&` (ԵՎ), `!` (ՈՉ), `??` (Null-ի հետ համակցման). Այստեղ մենք կծանոթանանք առաջին երեքին։ `??` օպերատորին կարող եք ծանոթանալ հաջորդ հոդվածում։
 
-Although they are called "logical", they can be applied to values of any type, not only boolean. Their result can also be of any type.
+Չնայած նրանք անվանվում են "տրամաբանական", նրանք կարող են կիրառվել կամայական(տարբերվող տրամաբանական տիպից) տիպի վրա։ Նրանց արդյունքը նույպես կարող է լինել կամայական տիպի։
 
-Let's see the details.
+Եկեք դիտարկենք ավելի մանրամասն։
 
-## || (OR)
+## || (ԿԱՄ) (OR)
 
-The "OR" operator is represented with two vertical line symbols:
+"ԿԱՄ" օպերատորը ներկայացվում է երկու ուղղահայաց գիծ նշանների միջոցով։
 
 ```js
-result = a || b;
+result = a || b
 ```
 
-In classical programming, the logical OR is meant to manipulate boolean values only. If any of its arguments are `true`, it returns `true`, otherwise it returns `false`.
+Դասական ծրագրավորման մեջ տրամաբանական ԿԱՄ օպերատորը կիրառվում է միայն տրամաբանական արժեքների վրա։ Եթե նրա արգումենտներից մեկը ճշմարի `true` է, ապա այն վերադարձնում է `true`, հակառակ դեպքում՝ `false`։
 
-In JavaScript, the operator is a little bit trickier and more powerful. But first, let's see what happens with boolean values.
+JavaScript֊ում այս օպերատորը ավելի ճկուն և հզոր է։ Բայց սկզում եկեք տեսնենք, թե ինչ է կատարվում տրամաբանական տիպերի հետ։
 
-There are four possible logical combinations:
+Կան չորս տրամաբանական կոմբինայիաներ․
 
 ```js run
-alert( true || true );   // true
-alert( false || true );  // true
-alert( true || false );  // true
-alert( false || false ); // false
+alert(true || true) // true
+alert(false || true) // true
+alert(true || false) // true
+alert(false || false) // false
 ```
 
-As we can see, the result is always `true` except for the case when both operands are `false`.
+Ինչպես տեսնում ենք արդյունքը միշտ ճիշտ(`true`) է, բացի այն դեպքից, երբ երկու օպերանդներն էլ սխալ(`false`) են։
```

Comment:
Please put spaces before parentheses. @vahmelk99 make sure to check other places too.